style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Raleway", sans-serif;
+  background-color: #344a72;
+  color: #fff;
+  line-height: 1.8;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
```

Comment:
Hello @bernyhas ,

You almost done and looking great your project  in fact   can you improve your base css.

Cheers!